saleor/dashboard/order/forms.py

```diff
@@ -356,3 +358,8 @@ def save(self):
         add_variant_to_delivery_group(
             self.group, variant, quantity, self.discounts)
         recalculate_order(self.group.order)
+
+
+class DashboardAddressForm(AddressForm):
```

Comment:
We already know that this is a "dashboard" form by the path of this module, so there's no need to mention that in the name. If you're trying to solve the problem of conflicting names, you should rather change the name of the imported `AddressForm`:
```from ...userprofile.i18n import AddressForm as StorefrontAddressForm```

Reply:
Damn, I completely forgot about `import ... as ...`, thank you